source/how-we-work/how-to-run-a-design-system-team-ceremony-or-workshop.html.md

```diff
@@ -0,0 +1,165 @@
+---
+title: How to run a Design System team ceremony or workshop
+weight: 5
+---
+
+# How to run a Design System team ceremony or workshop 
+
+
+This document details the principles, tools and resources we use when planning and running a ceremony or workshop on the Design System team.
+## Principles
+
+There are some principles that are important to remember when planning and facilitating a ceremony or workshop on the Design System team;
+
+- consider if there are asynchronous methods you can use to get the information you need. With remote working it is easy to experience video call fatigue.
+- share an agenda and session details ahead of time (including links to slide decks and collaboration tools). This helps participants prepare for the session and frame their thoughts - it can be difficult to think on the spot.
+- consider the accessibility of the tools used in the session - we prefer to use Trello or Padlet.
+- make use of the ‘optional’ invite for those who don’t need to be there
+- if it’s an open workshop, consider adding the meeting to the ‘Design System team events’ calendar which can be seen by all.
```

Comment:
```suggestion
- if it’s an open workshop, consider adding the meeting to the Design System team events calendar which can be seen by all. Ask a delivery manager for a link to the calendar.
```

Reply:
Don't think we can share the calendar openly, even though it's access-controlled, so adding this as a workaround.